spring-cloud-gcp-bigquery/src/main/java/org/springframework/cloud/gcp/bigquery/integration/outbound/BigQueryFileMessageHandler.java

```diff
@@ -168,7 +130,7 @@ public void setFormatOptions(FormatOptions formatOptions) {
 	 */
 	public void setWriteDisposition(WriteDisposition writeDisposition) {
```

Comment:
I don't think you need this method anymore and the two below, because they can just set it directly on the template. You can create a getter for the template if you want.

Reply:
Done, removed.